@xen-orchestra/xapi/vm.mjs

```diff
@@ -599,6 +599,32 @@ class Vm {
     }
   }
 
+  async leafCoalesce($defer, vmRef) {
```

Comment:
Why not `coalesceLeaf`?

It seems much more logical to me to have a `startVm` function than a `vmStart`.

Reply:
It was to keep the same name as the plugin function

Reply:
No, it's the name of the plugin (`leaf-coalesce`), not the function (`coalesce-leaf`).

And between both names, I think the last one is the least surprising.